src/iomusxml.cpp

```diff
@@ -2846,51 +2848,24 @@ void MusicXmlInput::ReadMusicXmlNote(
         // articulation
         std::vector<data_ARTICULATION> artics;
         for (pugi::xml_node articulations : notations.node().children("articulations")) {
-            if (notations.node().select_node("articulations/*[not(@placement)]")) {
+            for (pugi::xml_node articulation : articulations.children()) {
                 Artic *artic = new Artic();
-                for (pugi::xml_node articulation : articulations.children()) {
-                    artics.push_back(ConvertArticulations(articulation.name()));
-                    if (!std::strcmp(articulation.name(), "detached-legato")) {
-                        artics.push_back(ARTICULATION_stacc);
-                        artics.push_back(ARTICULATION_ten);
-                    }
+                artics.push_back(ConvertArticulations(articulation.name()));
+                if (!std::strcmp(articulation.name(), "detached-legato")) {
+                    artics.clear();
+                    artics.push_back(ARTICULATION_stacc);
+                    artics.push_back(ARTICULATION_ten);
```

Comment:
It seems that you are still setting multiple values to `@artic` attributes. So I assume this is still work-in-progress. Correct?